packages/3dt/build

```diff
@@ -36,7 +36,7 @@ EOF
 
 cat <<EOF | tee $master_service
 [Unit]
-Description=Diagnostics: DC/OS Distributed Diagnostics Tool Master API and Aggregation Service
+Description=DC/OS Diagnostics Master: diagnostics aggregation service
```

Comment:
how do you recommend saying that concisely? There's two different behaviors being performed: 
- service status aggregation - on every node
- node + service status aggregation - on the master nodes + exposing an API

The previous text didn't really say that to me